src/Modules/CalcDefence.lua

```diff
@@ -89,8 +89,23 @@ function calcs.reducePoolsByDamage(poolTable, damageTable, actor)
 	local output = actor.output
 	local modDB = actor.modDB
 	local poolTbl = poolTable or { }
-	local frostShield = poolTbl.FrostShieldLife or output.FrostShieldLife or 0
-	local soulLink = poolTbl.SoulLink or output.SoulLink or 0
+	
+	local alliesTakenBeforeYou = poolTbl.alliesTakenBeforeYou
+	if not alliesTakenBeforeYou then
+		alliesTakenBeforeYou = {}
+		if output.FrostShieldLife then
```

Comment:
Should frost shield be grouped inside a allies take damage before you array?

Reply:
frost shield is its own entity and doesnt count as you taking the damage, hence being grouped as a "non-you" entity